### DISCUSSION TITLE
How can I enable MX for Citus?

### DISCUSSION THREAD

**Discussion by kallydev** [citusdata/citus#5600] [Q&A] [answered]
Hi everyone, I learned in older versions of the documentation that Citus supports MX, this feature is very useful for me, but I didn't find any information about MX in the latest documentation.

How can I enable MX for Citus? Is Citus MX currently available?

https://docs.citusdata.com/en/v9.3/arch/mx.html

**Comment by marcocitus** [comment, marked as answer]
You can enable metadata syncing (on 10.2 or lower) via:
```
SELECT start_metadata_sync_to_node(nodename, nodeport) FROM pg_dist_node;
```

On the master branch, metadata syncing is enabled by default.
